src/workloads/query/TimeSeriesLastpoint.yml

```diff
@@ -53,369 +57,330 @@ LastpointQueryMetaAscTimeAsc: &LastpointQueryMetaAscTimeAsc
+          Operation:
+            OperationName: RunCommand
+            OperationCommand: {drop: *coll}
+
   - Name: CreateTimeSeriesCollection
     Type: RunCommand
     Threads: 1
     Phases:
-      - Repeat: 1
-        Database: *db
-        Operation:
-          OperationMetricsName: CreateTimeSeriesCollection
-          OperationName: RunCommand
-          OperationCommand: {create: *coll, timeseries: {timeField: "timestamp", metaField: "metadata"}}
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
+      OnlyActiveInPhases:
+        Active: [1]
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          Repeat: 1
+          Database: *db
+          Operation:
+            OperationMetricsName: CreateTimeSeriesCollection
+            OperationName: RunCommand
+            OperationCommand: {create: *coll, timeseries: {timeField: "timestamp", metaField: "metadata"}}
 
   - Name: InsertData
     Type: Loader
     Threads: 1
     Phases:
-      - *Nop
-      - Repeat: 1
-        Database: *db
-        Collection: *coll
-        Threads: 1
-        CollectionCount: 1
-        DocumentCount: 1000000
-        BatchSize: 100
-        Document:
-          timestamp: {^RandomDate: {min: "2022-01-01", max: "2022-03-01"}}
-          metadata: {sensorId: {^RandomInt: {min: 0, max: 100}}, type: "temperature"}
-          temp: {^RandomDouble: {min: -30, max: 120}}
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
-      - *Nop
+      OnlyActiveInPhases:
+        Active: [2]
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          Repeat: 1
+          Database: *db
+          Collection: *coll
+          Threads: 1
+          CollectionCount: 1
+          DocumentCount: 1_000_000
+          BatchSize: 100
+          Document:
+            timestamp: {^RandomDate: {min: "2022-01-01", max: "2022-03-01"}}
+            metadata: {sensorId: {^RandomInt: {min: 0, max: 100}}, type: "temperature"}
+            temp: {^RandomDouble: {min: -30, max: 120}}
+
+  # QuiesceActor will run in phase 3.
 
   # Lastpoint query with a sort and group with meta subfield descending and time ascending, but no index.
   - Name: RunLastpointQueryWithMetaSubfieldAscendingTimeDescendingNoIndex
+    Type: CrudActor
+    Database: *db
+    Phases:
+      OnlyActiveInPhases:
+        Active: [4]
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          *LastpointQueryMetaAscTimeDesc
+
+  - Name: CreateIdxForMetaAscTimeDesc
```

Comment:
Separated out CreateIndex and DropIndex actors so that aggregate can use `CrudActor`. Now `phases/query/TimeSeriesLastpoint.yml` is shared between TimeSeriesLastpoint.yml and TimeSeriesLastpointHighCardinality.yml in #1218 and so, `CrudActor` is necessary to get the complete results for high cardinality workload.